src/lib/validator/GameCommand.ts

```diff
@@ -0,0 +1,30 @@
+import ValidationError from '../error/ValidationError';
+import { ERROR_MESSAGE } from '../../constants/Messages';
+
+const GameCommand = {
+  checkGameCommand(input: unknown) {
+    this.checkNumber(input);
+    this.checkLength(input as string);
+    this.checkOneOrTwo(input as string);
+  },
+
+  checkNumber(input: unknown) {
+    if (/\D/.test(input as any)) {
```

Comment:
들어오는 input이 string으로 고정되어있다고 생각하기때문에, 해당 타입을 unknown에서 string으로 수정한다면
아래에 나오는 as any의 사용도 피할 수 있을 것 같아요.

** 위의 보리님과의 대화도 한번 읽어봤는데, 저 또한 타입을 `string`으로 고정하는 것이 더 안전하다고 생각해요!

```suggestion
  checkNumber(input: string) {
    if (/\D/.test(input)) {
```